website_sale_product_minimal_price/static/src/js/website_sale_product_price_scale.js

```diff
@@ -0,0 +1,84 @@
+odoo.define("website_sale_product_minimal_price.load", function (require) {
+    "use strict";
+    const ajax = require("web.ajax");
+    const core = require("web.core");
+    const field_utils = require("web.field_utils");
+    const publicWidget = require("web.public.widget");
+    const VariantMixin = require("sale.VariantMixin");
+    const QWeb = core.qweb;
+    const load_xml = ajax.loadXML(
+        "/website_sale_product_minimal_price/static/src/xml/" +
+            "website_sale_product_price_scale.xml",
+        QWeb
+    );
```

Comment:
You'll need this (tour is failing but you can go on debuggin form here :))
```suggestion
    );
    require("website_sale.website_sale");
```

Reply:
Right, that was the missing piece, thanks :)